src/Prop/Heading.js

```diff
@@ -70,16 +72,15 @@ type PropTypeHeadingProps = {
 
 const PropTypeHeading = (props: PropTypeHeadingProps) => (
   <Heading>
-    <code>
       <HeadingName>{props.name}</HeadingName>
+      <Whitespace />
```

Comment:
this could use JSX space as well if you don't want to use component here  `{' '}`